### PR TITLE
docs: add 4.12.0 changelog

### PR DESCRIPTION
**Pull request by chenshuai2144**
## 4.12.0

- 🆕 InputNumber 添加 `keyboard` 属性以支持禁用键盘行为。[#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
- 💄 修复 Row 配置垂直 `gutter` 时会额外添加 `margin-bottom` 样式的问题。[#29059](https://github.com/ant-design/ant-design/pull/29059)
- 🐞 修复 Input 1/2 不能输入的问题[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
- ⚡️ 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
- 🌐 为表格添加 en-GB 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
- Badge
  - 🐞 修复 Badge 的数字动画问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
  - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
- Table
  - 🆕 Table showSorterTooltip 属性支持 Tooltip 的配置。[#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
  - ⚡️ 优化 Table `rowSelection` 中 `onChange` 和 `onSelectAll` 的调用顺序。[#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
- TypeScript
  - 🤖 Table ts 定义更新 `dataSource` 至 `readonly`。[#29084](https://github.com/ant-design/ant-design/pull/29084)

-------------

## 4.12.0

- 🆕 InputNumber add `keyboard` prop to support disable keyboard behavior. [#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
- 💄 Fix Row with vertical `gutter` provides additional `margin-bottom` style. [#29059](https://github.com/ant-design/ant-design/pull/29059)
- 🐞 Fix the problem that Input 1/2 cannot be entered [#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
- ⚡️ Optimize ConfigProvider to avoid import all icons when tree shaking is not enabled. [#29045](https://github.com/ant-design/ant-design/pull/29045)
- 🌐 Add en-GB missing translations for table. [#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
- Table
  - 🆕 The Table showSorterTooltip property supports the configuration of Tooltip. [#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
  - ⚡️ Optimize Calling sequence of `onChange` and `onSelectAll` in `rowSelection` of Table. [#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
- Badge
  - 💄 Fix Badge change `count` content to hide miss style. [#29089](https://github.com/ant-design/ant-design/pull/29089)
  - 🐞 Fix Badge's digital animation problem. [#29100](https://github.com/ant-design/ant-design/pull/29100)
- TypeScript
  - 🤖 Table ts definition update `dataSource` to `readonly`. [#29084](https://github.com/ant-design/ant-design/pull/29084)
-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/changelog-4.11.3/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/changelog-4.11.3/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,15 @@ timeline: true
 
 ---
 
+## 4.11.3
+
+- ⚡️ 优化表格 `rowSelection` 中 `onChange` 和 `onSelectAll` 的调用顺序。[#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
+- 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
+- 🐞 Fix English grammar of Affix page。[#29069](https://github.com/ant-design/ant-design/pull/29069) [@renny-ren](https://github.com/renny-ren)
```
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:22]: 中文？

Comment (afc163) [reply CHANGELOG.zh-CN.md:22]: 文档改动不必写。

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/29122

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -19,8 +19,9 @@ timeline: true
 
 - ⚡️ 优化表格 `rowSelection` 中 `onChange` 和 `onSelectAll` 的调用顺序。[#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
 - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
-- 🐞 Fix English grammar of Affix page。[#29069](https://github.com/ant-design/ant-design/pull/29069) [@renny-ren](https://github.com/renny-ren)
 - ⚡️ 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🐞 修复 Badge 的数字动画问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
+- 🌐 为表格添加 en-GB 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🌐 Table 补充 en-GB 语言包。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
```

File: CHANGELOG.zh-CN.md [hunk 3]
```diff
@@ -19,8 +19,9 @@ timeline: true
 
 - ⚡️ 优化表格 `rowSelection` 中 `onChange` 和 `onSelectAll` 的调用顺序。[#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
 - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
-- 🐞 Fix English grammar of Affix page。[#29069](https://github.com/ant-design/ant-design/pull/29069) [@renny-ren](https://github.com/renny-ren)
 - ⚡️ 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🐞 修复 Badge 的数字动画问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]: Badge 组件放在一个分类下。

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "antd",
-  "version": "4.11.2",
+  "version": "4.11.3",
```
Comment (zombieJ) [inline package.json:3]: 月底了 `4.12.0`?

Comment (chenshuai2144) [reply package.json:3]: 提前发了 11 了，一个月两个大版本？

File: CHANGELOG.en-US.md [hunk 5]
```diff
@@ -15,6 +15,22 @@ timeline: true
 
 ---
 
+## 4.12.0
+
+- 🆕 InputNumber add `keyboard` prop to support disable keyboard behavior. [#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
+- 💄 Fix Row with vertical `gutter` provides additional `margin-bottom` style. [#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 Fix the problem that Input 1/2 cannot be entered [#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
```
Comment (zombieJ) [inline CHANGELOG.en-US.md:22]: 这个有点语焉不详，应该是输入 emoji 时，`showCount` 展示数据不正确的问题

**Comment by xrkffgg** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/29105
- [ ] https://github.com/ant-design/ant-design/pull/28981

File: CHANGELOG.en-US.md (hunk 5 above)
Comment (chenshuai2144) [reply CHANGELOG.en-US.md:22]: ok，我看到问题有点美哦理解

**Comment by chenshuai2144** [comment]
这个两个估计合不进去，ci 都挂了

**Comment by afc163** [comment]
rc-input-number 需要升级上来。

**Comment by shaodahong** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/29153

File: CHANGELOG.zh-CN.md [hunk 6]
```diff
@@ -15,6 +15,22 @@ timeline: true
 
 ---
 
+## 4.12.0
+
+- 🆕 InputNumber 添加 `keyboard` 属性以支持禁用键盘行为。[#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
+- 💄 修复 Row 配置垂直 `gutter` 时会额外添加 `margin-bottom` 样式的问题。[#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 修复输入 emoji 时，showCount 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
+- ⚡️ 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🌐 为表格添加 en-GB 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:24]:
```suggestion
- 🇬🇧 为表格添加 `en-GB` 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
```

File: CHANGELOG.zh-CN.md [hunk 7]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+- Badge
+  - 🐞 修复 Badge 的数字动画问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
+  - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
+- Table
+  - 🆕 Table showSorterTooltip 属性支持 Tooltip 的配置。[#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:29]:
```suggestion
  - 🆕 Table `showSorterTooltip` 属性支持 Tooltip 的配置。[#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
```

File: CHANGELOG.zh-CN.md [hunk 8]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
 
+## 4.12.0
+
+- 🆕 InputNumber 添加 `keyboard` 属性以支持禁用键盘行为。[#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
+- 💄 修复 Row 配置垂直 `gutter` 时会额外添加 `margin-bottom` 样式的问题。[#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 修复输入 emoji 时，showCount 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
+- ⚡️ 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🌐 为表格添加 en-GB 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
+- Badge
+  - 🐞 修复 Badge 的数字动画问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:26]:
```suggestion
  - 🐞 修复 Badge 的数字动画方向问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
```

File: CHANGELOG.zh-CN.md [hunk 9]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+  - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
+- Table
+  - 🆕 Table showSorterTooltip 属性支持 Tooltip 的配置。[#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
+  - ⚡️ 优化 Table `rowSelection` 中 `onChange` 和 `onSelectAll` 的调用顺序。[#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
+- TypeScript
+  - 🤖 Table ts 定义更新 `dataSource` 至 `readonly`。[#29084](https://github.com/ant-design/ant-design/pull/29084)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:32]:
```suggestion
  - 🤖 Table TS 定义更新 `dataSource` 至 `readonly`。[#29084](https://github.com/ant-design/ant-design/pull/29084)
```

File: package.json (hunk 4 above)
Comment (afc163) [reply package.json:3]: 标题是 4.12.0，这里是不是要统一一下。

**Comment by afc163** [comment]
feature 分支上有新东西了，需要合并进来。

File: CHANGELOG.en-US.md [hunk 10]
```diff
@@ -15,6 +15,22 @@ timeline: true
 
 ---
 
+## 4.12.0
+
+- 🆕 InputNumber add `keyboard` prop to support disable keyboard behavior. [#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
```
Comment (afc163) [inline CHANGELOG.en-US.md:20]:
```suggestion
- 🆕 InputNumber add `keyboard` prop which could disable keyboard behavior. [#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
```

File: CHANGELOG.en-US.md [hunk 11]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+- 💄 Fix Row with vertical `gutter` provides additional `margin-bottom` style. [#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 Fix the issue that showCount displays incorrect data when emoji is input. [#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
+- ⚡️ Optimize ConfigProvider to avoid import all icons when tree shaking is not enabled. [#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🌐 Add en-GB missing translations for table. [#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
+- Table
+  - 🆕 The Table showSorterTooltip property supports the configuration of Tooltip. [#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
```
Comment (afc163) [inline CHANGELOG.en-US.md:26]:
```suggestion
  - 🆕  Table `showSorterTooltip` now could be a object for more configuration. [#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
```

File: CHANGELOG.en-US.md [hunk 12]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+  - ⚡️ Optimize Calling sequence of `onChange` and `onSelectAll` in `rowSelection` of Table. [#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
```
Comment (afc163) [inline CHANGELOG.en-US.md:27]:
```suggestion
  - ⚡️ Optimize calling sequence of `onChange` and `onSelectAll` in `rowSelection` of Table. [#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
```

File: CHANGELOG.en-US.md [hunk 13]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+- Badge
+  - 💄 Fix Badge change `count` content to hide miss style. [#29089](https://github.com/ant-design/ant-design/pull/29089)
+  - 🐞 Fix Badge digital animation problem. [#29100](https://github.com/ant-design/ant-design/pull/29100)
+- TypeScript
+  - 🤖 Table ts definition update `dataSource` to `readonly`. [#29084](https://github.com/ant-design/ant-design/pull/29084)
```
Comment (afc163) [inline CHANGELOG.en-US.md:32]:
```suggestion
  - 🤖 Table TS definition update `dataSource` to `readonly`. [#29084](https://github.com/ant-design/ant-design/pull/29084)
```

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/29144

**Comment by afc163** [comment]
- [ ] https://github.com/ant-design/ant-design/pull/29105

File: CHANGELOG.zh-CN.md [hunk 14]
```diff
@@ -15,6 +15,22 @@ timeline: true
 
 ---
 
+## 4.12.0
+
+- 🆕 InputNumber 添加 `keyboard` 属性以支持禁用键盘行为。[#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
+- 💄 修复 Row 配置垂直 `gutter` 时会额外添加 `margin-bottom` 样式的问题。[#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 修复输入 emoji 时，showCount 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:22]:
```suggestion
- 🐞 修复 Textarea 输入 emoji 时 `showCount` 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
```

File: CHANGELOG.zh-CN.md [hunk 15]
```diff
@@ -15,6 +15,22 @@ timeline: true
...
+- ⚡️ 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:23]:
```suggestion
- 📦 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
```

File: CHANGELOG.zh-CN.md (hunk 14 above)
Comment (afc163) [reply CHANGELOG.zh-CN.md:22]: 英文记得也改下。

File: CHANGELOG.zh-CN.md (hunk 15 above)
Comment (afc163) [reply CHANGELOG.zh-CN.md:23]: 英文记得也改下。

File: CHANGELOG.zh-CN.md [hunk 16]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+- 📦 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🇬🇧 为表格添加 `en-GB` 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
+- Badge
+  - 🐞 修复 Badge 的数字动画方向问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
+  - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
+- Table
+  - 🆕 Table `showSorterTooltip` 属性支持 Tooltip 的配置。[#29002](https://github.com/ant-design/ant-design/pull/29002) [@harrisoff](https://github.com/harrisoff)
+  - ⚡️ 优化 Table `rowSelection` 中 `onChange` 和 `onSelectAll` 的调用顺序。[#29079](https://github.com/ant-design/ant-design/pull/29079) [@kerm1it](https://github.com/kerm1it)
+- TypeScript
+  - 🤖 Table Typescript 定义更新 `dataSource` 至 `readonly`。[#29084](https://github.com/ant-design/ant-design/pull/29084)
```
Comment (xrkffgg) [inline CHANGELOG.zh-CN.md:37]:
```suggestion
  - 🤖 更新 Table Typescript 定义 `dataSource` 至 `readonly`。[#29084](https://github.com/ant-design/ant-design/pull/29084)
```

File: CHANGELOG.zh-CN.md [hunk 17]
```diff
@@ -15,6 +15,27 @@ timeline: true
...
+## 4.12.0
+
+- 🆕 Image.PreviewGroup 添加 `current` 属性支持受控模式。[#29153](https://github.com/ant-design/ant-design/pull/29153)
+- InputNumber
+  - 🆕 InputNumber 支持 `bordered` 属性。[#29105](https://github.com/ant-design/ant-design/pull/29105)
+  - 🆕 InputNumber 添加 `keyboard` 属性以支持禁用键盘行为。[#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
+- 💄 加深 Tag 文字颜色以提升可读性。[#29144](https://github.com/ant-design/ant-design/pull/29144)
+- 💄 修复 Row 配置垂直 `gutter` 时会额外添加 `margin-bottom` 样式的问题。[#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 修复 Collapse 标题区域隐藏延迟的问题。[#29158](https://github.com/ant-design/ant-design/pull/29158)
+- 🐞 修复 Textarea 输入 emoji 时 `showCount` 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
```
Comment (AlanCutFlim) [inline CHANGELOG.zh-CN.md:27]:
```suggestion
- 🐞 修复 TextArea 输入 emoji 时 `showCount` 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
```

File: CHANGELOG.en-US.md [hunk 18]
```diff
@@ -15,6 +15,27 @@ timeline: true
 
 ---
 
+## 4.12.0
+
+- 🆕 Image.PreviewGroup add `current` prop. [#29153](https://github.com/ant-design/ant-design/pull/29153)
+- InputNumber
+  - 🆕 InputNumber support bordered prop. [#29105](https://github.com/ant-design/ant-design/pull/29105)
```
Comment (afc163) [inline CHANGELOG.en-US.md:22]:
```suggestion
  - 🆕 InputNumber support `bordered` prop. [#29105](https://github.com/ant-design/ant-design/pull/29105)
```

**Comment by afc163** [comment]
图片

页面布局挂了，我看看什么问题。

File: CHANGELOG.zh-CN.md [hunk 19]
```diff
@@ -15,6 +15,29 @@ timeline: true
...
+  - 🆕 InputNumber 添加 `keyboard` 属性以支持禁用键盘行为。[#29110](https://github.com/ant-design/ant-design/pull/29110) [@kerm1it](https://github.com/kerm1it)
+- 💄 加深 Tag 文字颜色以提升可读性。[#29144](https://github.com/ant-design/ant-design/pull/29144)
+- 💄 修复 Row 配置垂直 `gutter` 时会额外添加 `margin-bottom` 样式的问题。[#29059](https://github.com/ant-design/ant-design/pull/29059)
+- 🐞 修复 Collapse 标题区域隐藏延迟的问题。[#29158](https://github.com/ant-design/ant-design/pull/29158)
+- 🐞 修复 TextArea 输入 emoji 时 `showCount` 展示数据不正确的问题。[#29057](https://github.com/ant-design/ant-design/pull/29057) [@HomyeeKing](https://github.com/HomyeeKing)
+- 📦 优化 ConfigProvider 在未开启 tree shaking 的项目中引入所有图标的问题。[#29045](https://github.com/ant-design/ant-design/pull/29045)
+- 🇬🇧 为表格添加 `en-GB` 翻译。[#29131](https://github.com/ant-design/ant-design/pull/29131) [@eberjoe](https://github.com/eberjoe)
+- Badge
+  - 🐞 修复 Badge 的数字动画方向问题。[#29100](https://github.com/ant-design/ant-design/pull/29100)
+  - 💄 修复 Badge 修改 `count` 内容以隐藏时的样式问题。[#29089](https://github.com/ant-design/ant-design/pull/29089)
```
Comment (MrHeer) [inline CHANGELOG.zh-CN.md:32]: 为啥这两个的顺序和英文版的不一样？

**Pull request merged**